### DISCUSSION TITLE
ideia

### DISCUSSION THREAD

**Discussion by luiz1238** [alyssapiresfernandescefet/openrpg#10] [Ideas]
só queria saber se seria possível ter uma função na ficha do player dele adicionar as próprias coisas e rituais

**Comment by lucaspiresfernandes** [comment]
Como assim? O que você define como "coisas"? Até o momento, o player tem total liberdade de adicionar e apagar rituais, equipamentos e itens, contanto que essas informações tenham sido adicionadas pelo mestre.

**Comment by luiz1238** [comment]
eu digo justamente de ter os itens já colocados pelo mestre, e ter uma outra opção do player criar durante campanha

exemplo : um colar com algo paranormal nele, mas o mestre não colocou isso na parte de itens, o player iria lá e criava 

mais ou menos isso

**Comment by lucaspiresfernandes** [comment]
Infelizmente, não tenho nenhuma perspectiva para adicionar uma nova funcionalidade que permita o próprio jogador editar a ficha. Me desculpe, mas não é dessa forma que projetei o Open RPG, e ele é totalmente orientado ao mestre.

No entanto, já existe uma funcionalidade que pode ser interessante para esse caso de uso: o botão da visibilidade, disponível no editor para os equipamentos, magias e itens. Esse botão permite que você esconda e mostre informações para os jogadores, para talvez disponibilizar uma nova arma ou item na hora certa, durante a campanha.

O ideal é você, como mestre, registrar todas as armas/itens/magias disponíveis na sua campanha e manipular a disponibilidade delas para os jogadores de acordo com a necessidade.

**Comment by luiz1238** [comment]
tudo bem, obrigado por me explicar direitinho :)
